README.md

```diff
@@ -4,28 +4,33 @@
 
 ## What is Rafiki?
 
-Rafiki is an open source package that exposes a comprehensive set of
-Interledger APIs. It's intended to be run by wallet providers, allowing them to
-offer Interledger functionality to their users.
+Rafiki is an open source package that allows an [Account Servicing Entity](./docs/glossary.md#account-servicing-entity) to enable [Interledger](./docs/glossary.md#interledger-protocol) functionality on its users' accounts.
 
-Rafiki is made up of several components including an Interledger connector, a
-high-throughput accounting database, and an API which can be accessed directly
-by users to implement Interledger applications.
+This includes
 
-Rafiki also allows for delegated access, offering OAuth-based flows to grant
-third-party applications access to Interledger functionality on a user's
-account.
+- sending and receiving payments (via [SPSP](./docs/glossary.md#simple-payments-setup-protocol-spsp) and [Open Payments](./docs/glossary.md#open-payments))
+- allowing third-party access to initiate payments and view transation data (via [Open Payments](./docs/glossary.md#open-payments))
+
+**❗ Rafiki is intended to be run by [Account Servicing Entities](./docs/glossary.md#account-servicing-entity) only and must not be used in production by non-regulated entities.**
```

Comment:
This seems to indicate that non-regulated entities don't have permission.
Maybe
```suggestion
**❗ Rafiki is intended to be run by [Account Servicing Entities](./docs/glossary.md#account-servicing-entity) only and should not be used in production by non-regulated entities.**
```

Reply:
I'm not so good with these kind of nuances. Thanks!